composer/trainer/devices/device.py

```diff
@@ -10,15 +12,18 @@
 from composer.core.types import Batch, BatchPair, Optimizer, Precision, Tensor
 from composer.utils.iter_helpers import map_collection
 
+__all__ = ["Device", "T_nnModule"]
+
 T_nnModule = TypeVar("T_nnModule", bound=torch.nn.Module)
```

Comment:
I only exported it because it's used as a parameter to some things in device so I figured it's best to cross reference.

Reply:
That's fine; it would probably be better to re-define this in each file where it is used (I think it's rare for TypeVars to be re-imported across files)